### PR TITLE
tool to compare sources manifest and sbom

### PR DESCRIPTION
**Pull request by deitch**
Is not used in this PR, just lays the groundwork for it.

### CODE REVIEW

**Comment by deitch** [comment]
Don't merge this quite yet. I want to add the kernel packages to it.

File: tools/compare-sbom-sources/main.go [hunk 1]
```diff
@@ -0,0 +1,333 @@
+package main
```
Comment (eriknordmark) [inline tools/compare-sbom-sources/main.go:1]: Copyright and SPDX license?

**Review by eriknordmark — COMMENTED** [review]
I assume you'll add something to docs/ on how this can be used.

File: tools/compare-sbom-sources/main.go (hunk 1 above)
Comment (deitch) [reply tools/compare-sbom-sources/main.go:4]: Added

**Comment by deitch** [comment]
> I assume you'll add something to docs/ on how this can be used.

Added.

**Comment by deitch** [comment]
Ready for review. I am ignoring kernel for now, as there is no way to reconcile them. The structure is just too different, fundamentally.

File: docs/SBOM-AND-SOURCES.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+# SBoM and Sources
+
+eve-os comes with a [software bill of materials (SBoM)](https://en.wikipedia.org/wiki/Software_supply_chain)
```
Comment (eriknordmark) [inline docs/SBOM-AND-SOURCES.md:3]: Don't we capitalize this as EVE-OS elsewhere?

**Review by eriknordmark — APPROVED** [review]
LGTM except for the capitalization nit.

File: docs/SBOM-AND-SOURCES.md (hunk 2 above)
Comment (deitch) [reply docs/SBOM-AND-SOURCES.md]: I lost track. For a while we were doing it without, then with, or maybe that was just my impression.

I updated all references in this document to be EVE-OS.

**Comment by deitch** [comment]
> LGTM except for the capitalization nit.

Fixed

File: tools/compare-sbom-sources/main.go [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var defaultTypes = []string{"golang", "alpine"}
+
+type singlePackage struct {
+	name_version string
```
Comment (eriknordmark) [inline tools/compare-sbom-sources/main.go:23]: I didn't see this yetus failure earlier. Can you please change to camelCase?

**Pull request merged**